### PR TITLE
Reverts #25805 and #25846: "Only one IV bag at a time "

### PR DESCRIPTION
**Pull request by chuga-git**
## What Does This PR Do
Reverts #25805 and #25846 completely.

## Why It's Good For The Game

25805 was moderately controversial in balance team discussions before its merge for the following reasons:
1. The PR was intended as a patch for player behavior only seen on downstream. We had not observed multiple IV bags being abused in the manner described by the author on Paradise upstream.
2. The changes introduced a seriously annoying anti-QOL quirk for medbay players. Doctors are forced to painstakingly rifle through their patient's bags, pockets, and containers to find remove an IV bag before being able to insert a new one. This is _really_ annoying! 
3. I am personally not convinced this tradeoff is worth it for the sake of a niche balance issue.

## Testing

Stuck a bunch of IV bags in my body.
<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>
image

## Changelog

:cl: Chuga
tweak: You can now have more than one IV bag inserted at a time again
/:cl:

### CODE REVIEW

**Review by DGamerL — APPROVED** [review]
Revert LGTM, approval message could show a bit more that it's approving of your revert

**Comment by Danchi299** [comment]
Reverts completely?
I do like the part where is shows if person has an IV bag attached!

**Comment by Danchi299** [comment]
> Doctors are forced to painstakingly rifle through their patient's bags, pockets, and containers to find remove an IV bag before being able to insert a new one.

I don't understand this. at what point does a person in medbay has a blood IV in any of their storages while also being moved to operating room where they need new IV attached?

You know what, if this problem DOES arise more often that I met it myself,
why not just make it so new IV insertion overrides the old one?
it's really simple, as you try to insert new IV, remove the old one's attachment instead of giving an error message.
it prevents the possibility of people powergaming AND makes it  so you don't have to look through someone's pockets.

...

or just strip a person and move them 2 tiles so that IV rips from their arm, anti-vox approved.

**Comment by tipoima** [comment]
> 
> I don't understand this. at what point does a person in medbay has a blood IV in any of their storages while also being moved to operating room where they need new IV attached?
> 

Antags and gamers with IV drug pipelines, vampire victims who were given blood bags and proceeded to put them in a pocket before passing out

File: code/modules/reagents/reagent_containers/iv_bag.dm [hunk 1]
```diff
@@ -41,21 +41,12 @@
 	..()
 	update_icon(UPDATE_OVERLAYS)
 
-/obj/item/reagent_containers/iv_bag/proc/on_examine(datum/source, mob/examiner, list/examine_list)
-	SIGNAL_HANDLER // COMSIG_PARENT_EXAMINE
-	examine_list += "<span class='notice'>[source.p_they(TRUE)] [source.p_have()] an active IV bag.</span>"
-
```
Comment (lewcc) [inline code/modules/reagents/reagent_containers/iv_bag.dm:47]: Yeah, any chance we could keep this? It'd save a moment here and there in medbay when juggling blood loss patients.

Comment (chuga-git) [reply code/modules/reagents/reagent_containers/iv_bag.dm:47]: That's not a bad idea, actually. I'll implement that now.

**Review by lewcc — APPROVED** [review]

**Pull request merged**